src/Cart/CartProduct.js

```diff
@@ -0,0 +1,237 @@
+    marginLeft: 5,
+    marginTop: 25,
+  },
+  itemQuantity: {
+    fontFamily: 'Verdana',
+    fontStyle: 'normal',
+    fontWeight: 'normal',
+    fontSize: 14,
+    alignContent: 'center',
+  },
+  itemTotalPrice: {
+    fontFamily: 'Verdana',
+    fontStyle: 'normal',
+    fontWeight: 'bold',
+    fontSize: 18,
+    marginBottom: '0%',
+    marginLeft: 100,
+    marginTop: 10,
+  },
+  itemPricePer: {
+    fontFamily: 'Verdana',
+    fontStyle: 'normal',
+    fontWeight: 'normal',
+    fontSize: 12,
+    marginBottom: 0,
+    marginLeft: 100,
+  },
+  removeItemButton: {
+    width: '5%',
+    padding: 0,
+    marginTop: 15,
+    marginLeft: 135,
+    marginRight: 10,
+  },
+  image: {
+    width: 80,
+    height: 80,
+    marginLeft: 5,
+    marginTop: 20,
+    marginBottom: 20,
+  },
+});
+
+export default function CartProduct(props) {
+  const {
+    itemID,
+    refresh,
+    setRefresh,
+    name,
+    price,
+    type,
+    initialQuantity,
+  } = props;
+
+  const [item, setItem] = useState([]);
+  const [quantity, setQuantity] = React.useState(String(initialQuantity));
+
+  const getItem = () => {
+    base('Cart').select({ maxRecords: 1, filterByFormula: `({item_id}='${itemID}')` }).firstPage()
+      .then((records) => {
+        const newItem = records[0].fields;
+        setItem(newItem);
+        setQuantity(String(newItem.quantity));
+        console.log(`new item${newItem.quantity}`);
+      });
+  };
+
+  const handleQuantityChange2 = (newQuantity) => {
+    setQuantity(newQuantity);
+    console.log(`QUANTITY GOT CHANGED${quantity}`);
+    base('Cart').update([
+      {
+        id: String(itemID),
+        fields: {
+          quantity: Number(newQuantity),
+        },
+      },
+    ], (err, records) => {
+      if (err) {
+        console.error(err);
+        return;
+      }
+      records.forEach((record) => {
+        console.log(record.get('users'));
+      });
+    });
+    setRefresh(refresh + 1);
+  };
+
+  useEffect(() => {
+    getItem();
+  }, []);
+
+  const deleteItem = () => {
+    base('Cart').destroy([itemID], (err, deletedRecords) => {
+      if (err) {
+        console.error(err);
+        return;
+      }
+      console.log('Deleted', deletedRecords.length, 'records');
+    });
+    setRefresh(refresh + 1);
+  };
+  // yay product
+  return (
+    <View style={styles.container}>
+      <Image style={styles.image} source={image} />
+      <View style={styles.container3}>
+        <Text style={styles.itemName}>
+          {name}
+        </Text>
+        <View style={styles.quantityContainer}>
+          <TextInput
+            keyboardType="numeric"
+            value={quantity}
+            style={styles.quantityBox}
+            onChangeText={handleQuantityChange2}
+          />
+          <Text style={styles.itemQuantityType}>
+            {type}
+          </Text>
+        </View>
+      </View>
+      <View style={styles.container2}>
+        <Text style={styles.itemTotalPrice}>
+          $
```

Comment:
You can do this instead:           {`$ ${price * quantity}`} for all expressions combining string literals with expressions

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
like replace 
`<Text style={styles.itemTotalPrice}>
          $
          {(price) * quantity}
        </Text>`
        with `<Text style={styles.itemTotalPrice}>
           {$ ${price * quantity}} 
        </Text>`?